### DISCUSSION TITLE
Computer power requirements

### DISCUSSION THREAD

**Discussion by kieran** [cyclingwithelephants/devs-on-a-boat#8] [General]
Typical 27" monitors use an avg of ~30 W when on to a reasonable brightness.

MacBook Air (M1) uses about ~25 W avg when powered on, probably closer to 10W with the screen closed. Might draw 30-40 W when charging, ships with a 30W power supply but may use more when plugged into a PD supply that supports faster charging.

Mac Mini (M1) is reported to use 6.8 W idle, with a max draw of 39 W under full load. An M1 Air with a closed lid would likely be very similar, as they're effectively the same machine.

So if you use the monitor / macbook for 8h a day, that would be about ~440Wh usage. Realistically, one ~350W solar panel for 2 hours would probably comfortably cover this load.

**Comment by cyclingwithelephants** [comment]
This is pretty awesome, I have to say.

In a more general sense, how do you think we should approach solving this in general? As a starter for 10, I was thinking:
- We solve for the optimum case (as you have)
- we solve for the average case (e.g. couple year old hardware)
- we solve for a pessimistic case (old hardware, peak usage etc)

Any other considerations?
